trtools/testsupport/sample_vcfs/dumpSTR_vcfs/create_test_files.sh

```diff
@@ -0,0 +1,38 @@
+# Note: this command should be run from the root of the repo
+
+# Note: the thresholds in this file are arbitrary and just used for testing
+# do not assume they are reasonable for data analysis
+
```

Comment:
I think this is a really cool idea! Thanks for implementing it!

Reply:
Yeah! This took me quite a bit of effort, but it also caught a bunch of unintended changes that I made when I swapped the code over. So feels worth it.